app/assets/javascripts/views/HeaderView.js

```diff
@@ -1,116 +1,151 @@
 /**
  * The Header view.
  */
-define([
-  'jquery',
-  'backbone',
-  'underscore',
-  'mps',
-  'views/ShareView'
-], function($,Backbone, _,mps, ShareView) {
-
-  'use strict';
-
-  var HeaderView = Backbone.View.extend({
-
+define(['jquery', 'backbone', 'underscore', 'mps', 'views/ShareView'], (
+  $,
+  Backbone,
+  _,
+  mps,
+  ShareView
+) => {
```

Comment:
As the old app isnt part of the webpack app, we cannot have arrow functions here as it breaks internet explorer. Can you make sure any linter based changes to these files is reversed and add a lint exception?